packages/cli/src/oclif/commands/validate.js

```diff
@@ -9,10 +9,10 @@ const { validateApp } = require('../../utils/api');
 
 class ValidateCommand extends BaseCommand {
   async perform() {
-    this.log('Validating project locally.');
+    this.log('Validating project locally here.');
 
     const errors = await localAppCommand({ command: 'validate' });
-
+    this.log('Validating project locally: .', errors);
```

Comment:
this looks like a test string? since errors won't always be present and there's a trailing `.`? if so, you can remove.

Reply:
This was actually intentional sans the `.` and the `errors`. I'll clean it up.

Reply:
nvm nuked this